utils/constantine-2-genesis.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
```

Comment:
Thinking that should this just be a general type of script, so adding the files and burn accounts as variables?

Reply:
the library is supposed to be the general reference and this script applies only to this specific upgrade